test/README.md

```diff
@@ -33,29 +18,39 @@ Framework tests test the test framework itself.
 Running the test framework tests:
 
 ```bash
+kubectl -n "$STEWARD_TEST_NAMESPACE" delete secret --all
 ( cd framework && go test ./... -count=1 -tags=frameworktest -v -- --kubeconfig "$KUBECONFIG" )
 ```
 
 ## Running Tests
 
 ### Integration Tests
 
+Integratin tests are split into tree groups to avoid client-side throttling and too much load on the test cluster.
```

Comment:
Why not just removing `t.Parallel()` from the tests?

Reply:
See https://github.com/SAP/stewardci-core/pull/384/commits/cbbad39b103a481201043559e78ff1c85aae4c0d